### PR TITLE
fix cli config format

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: pages/docs/install/containers/docker-cloud.html [hunk 1]
```diff
@@ -30,16 +30,16 @@
 ```yaml
 crate1:
   image: 'crate:latest'
-  command: 'crate -Des.network.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: also add `-Cnetwork.bind_host=_site_`

File: pages/docs/install/containers/docker-cloud.html [hunk 2]
```diff
@@ -30,16 +30,16 @@
...
   ports:
     - '4200:4200'
     - '4300:4300'
 crate2:
   image: 'crate:latest'
-  command: 'crate -Des.network.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: same as above

File: pages/docs/install/containers/docker-cloud.html [hunk 3]
```diff
@@ -30,16 +30,16 @@
...
 crate3:
   image: 'crate:latest'
-  command: 'crate -Des.network.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: same as above

File: pages/docs/install/containers/docker.html [hunk 4]
```diff
@@ -47,15 +47,15 @@
 ```bash
 docker run -d -p 4200:4200 -p 4300:4300
   -v <data-dir>:/data crate
-  crate -Des.config=/path/to/crate.yml
-        -Des.cluster.name=cluster
+  crate -Cconfig=/path/to/crate.yml
```
Comment (celaus) [inline pages/docs/install/containers/docker.html]: `-Cconfig` became `-Cpath.conf` and points to a folder (containing the logging and general configuration

also the site binding thing needs to go into here as well (from above)

File: pages/docs/install/containers/docker.html [hunk 5]
```diff
@@ -74,7 +74,7 @@
     - /mnt/data/crate:/data
   environment:
     CRATE_HEAP_SIZE: 16g
-  command: crate -Des.config=/path/to/crate.yml -Des.cluster.name=cluster
+  command: crate -Cconfig=/path/to/crate.yml -Ccluster.name=cluster
```
Comment (celaus) [inline pages/docs/install/containers/docker.html]: same as above :)

File: pages/docs/scale/docker_swarm.html [hunk 6]
```diff
@@ -98,11 +98,11 @@
...
-    -Des.discovery.zen.ping.unicast.hosts=crate \
-    -Des.gateway.expected_nodes=3 \
-    -Des.discovery.zen.minimum_master_nodes=2 \
-    -Des.gateway.recover_after_nodes=2 \
-    -Des.network.bind=_eth0:ipv4_
+    -Cdiscovery.zen.ping.unicast.hosts=crate \
+    -Cgateway.expected_nodes=3 \
+    -Cdiscovery.zen.minimum_master_nodes=2 \
+    -Cgateway.recover_after_nodes=2 \
+    -Cnetwork.bind=_eth0:ipv4_
```
Comment (celaus) [inline pages/docs/scale/docker_swarm.html]: `-Cnetwork.bind=_site_`  instead of `_eth0:ipv4_`

File: pages/docs/scale/kubernetes.html [hunk 7]
```diff
@@ -48,12 +48,12 @@
         image: crate:latest
```
Comment (celaus) [inline pages/docs/scale/kubernetes.html]: to be honest the whole file doesn't really work anymore (k8s renamed petset to stateful set in 1.6, so it really works only in 1.5 with alpha API.... we need to rework this

Comment (nslater) [inline pages/docs/scale/kubernetes.html]: this will be addressed by https://trello.com/c/XoYd24o7/625

**Comment by nslater** [comment]
@celaus can you review my changes? thx!

File: pages/docs/install/containers/docker-cloud.html [hunk 8]
```diff
@@ -30,16 +30,16 @@
 ```yaml
 crate1:
   image: 'crate:latest'
-  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.bind_host=_site_ -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: `_ethwe:ipv4_` ?

File: pages/docs/install/containers/docker-cloud.html [hunk 9]
```diff
@@ -30,16 +30,16 @@
...
   ports:
     - '4200:4200'
     - '4300:4300'
 crate2:
   image: 'crate:latest'
-  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.bind_host=_site_ -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: same here

File: pages/docs/install/containers/docker-cloud.html [hunk 10]
```diff
@@ -30,16 +30,16 @@
...
 crate3:
   image: 'crate:latest'
-  command: 'crate -Cnetwork.publish_host=_ethwe:ipv4_'
+  command: 'crate -Cnetwork.bind_host=_site_ -Cnetwork.publish_host=_ethwe:ipv4_'
```
Comment (celaus) [inline pages/docs/install/containers/docker-cloud.html]: here

**Pull request merged**